test/shared.tsx

```diff
@@ -282,16 +291,25 @@ export function addInteractiveTests(render: typeof mount) {
         super(props);
       }
 
+      shouldComponentUpdate() {
+        return true;
+      }
+      componentWillReceiveProps() {}
+      componentWillMount() {}
+      componentDidMount() {}
+      componentDidUpdate() {}
+      componentWillUnmount() {}
+
       render() {
         return <div>Test</div>;
       }
     }
-    Test.prototype.shouldComponentUpdate = sinon.stub().returns(true);
-    Test.prototype.componentWillReceiveProps = sinon.stub();
-    Test.prototype.componentWillMount = sinon.stub();
-    Test.prototype.componentDidMount = sinon.stub();
-    Test.prototype.componentDidUpdate = sinon.stub();
-    Test.prototype.componentWillUnmount = sinon.stub();
+    sinon.stub(Test.prototype, 'shouldComponentUpdate').returns(true);
+    sinon.stub(Test.prototype, 'componentWillReceiveProps');
+    sinon.stub(Test.prototype, 'componentWillMount');
+    sinon.stub(Test.prototype, 'componentDidMount');
+    sinon.stub(Test.prototype, 'componentDidUpdate');
+    sinon.stub(Test.prototype, 'componentWillUnmount');
```

Comment:
While debugging some issues, I found stubbing using this method made test error messages more useful. Stubing this way, Sinon errors are like `'shouldComponentUpdate' was not called.` instead of of `stub was not called` and so made it easier to identity what the issue was. If you prefer the other way, I can revert.

Reply:
Ah, this is useful to know. I simply wasn't aware that Sinon could produce better errors with this approach.